types/jest-when/index.d.ts

```diff
@@ -23,7 +23,7 @@ export interface WhenMock<T = any, Y extends any[] = any>
   mockImplementationOnce(fn?: (...args: Y) => T): this;
 }
 
-export type When = <T, Y extends any[]>(fn: jest.MockInstance<T, Y>) => WhenMock<T, Y>;
+export type When = <T, Y extends any[]>(fn: ((...args: Y) => T) | jest.MockInstance<T, Y>) => WhenMock<T, Y>;
```

Comment:
Could this change be made to `jest.MockInstance` instead of just here?

Disclaimer: I don't know much about jest and nothing about jest-when.

Reply:
I looked at jest.MockInstance and it's a gigantic, complex type. So it's reasonable to add a call signature here.